src/v2i-hub/TelematicBridgePlugin/manifest.json

```diff
@@ -11,6 +11,31 @@
             "key": "LogLevel",
             "default": "INFO",
             "description": "The log level for this plugin"
+        },
+        {
+            "key": "NATSUrl",
+            "default": "nats://127.0.0.1:4222",
+            "description": "The log level for this plugin"            
+        },
+        {
+            "key": "UnitId",
+            "default": "v2xhub_id",
+            "description": "The log level for this plugin"            
+        },
+        {
+            "key": "UnitName",
+            "default": "East Intersection",
+            "description": "The log level for this plugin"            
+        },
+        {
+            "key": "UnitType",
+            "default": "Infrastructure",
```

Comment:
Not sure about this field.

Reply:
UnitType does not make sense as a configurable parameter if it is sent to infrastructre. Every V2X-Hub falls under infrastructure. Maybe if we have types like intersection or highway it would make sense but like this is does not make sense.

Reply:
This is a category defined by telematic tool. It only has two category platform and infrastructure. Carma-streets, carma-cloud, v2xhub belongs to infrastructure.

Reply:
So then lets make this a constant and not configurable.

Reply:
Addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/567/commits/d5998cf26339e7da466a751de298804391f2ad1c